Liður 1-tíðni nafna

snaeja13 opened the pull request:
Hér er kóðinn fyrir lið 1. Farið er inn í terminalið og keyrt skipunana
 ```sqlite3 data/names_freq.db < names.sql ``` til að fá niðurstöðuna.

snaejaa committed:
Lokaskil

Hér er dæmi 1

snaejaa committed:
útskýringin í README.md

fyrir dæmi 1

snaejaa committed:
README.md (dæmi1)

Hérna koma inn upplýsingar hvernig forrit og annað þarf til að leysa dæmi 1.

tungufoss commented:
Hér mætti vera meira lýsandi nafn, td endurspegla að þetta er liður 1 um tíðni nafna.

Líka hvernig eigi að keyra skrárnar, td endurtaka það sem þú settir í README

erlendur235 commented:
Sýnist allt vera í lagi

tungufoss commented:
> Sýnist allt vera í lagi

@erlendur235: Þú verður að velja "approve" - nýja reglan sem ég setti á verkefnin leyfa ekki merge á main nema 2 ólíkir samþykkja PR.

erlendur235 reviewed (dismissed):
Sýnist allt vera í lagi

erlendur235 reviewed (dismissed):
Já, flott svona

snaejaa committed:
Uppfærsla+útskyring koða

Hér þurfti ég að uppfæra eina villu í kóða þar sem ég setti óvart niðurstöðuna í vitlausa röð. Einnig set ég inn útskýringu með kóðanum

snaeja13 committed:
Merge branch 'main' into Dæmi_1

dalrosInga reviewed (dismissed):
allt komið núna, flott

snaejaa committed:
Breytt nafn

snaejaa committed:
Merge branch 'Dæmi_1' of https://github.com/Upplysingaverkfraedi/sqlite-baratheon into Dæmi_1

erlendur235 reviewed (approved):
Já mjög gott
